### PR TITLE
Feature/ci

### PR DESCRIPTION
**Pull request by davidkarlsen**
Adds CI capabilities using Github Actions.
It fails now due to the secret for anchore auth missing, but I can fix that in the refactoring branch.

---
This change is [Reviewable](https://reviewable.io/reviews/anchore/anchore-charts/14)

### CODE REVIEW

**Comment by zhill** [comment]
Great! Thanks @davidkarlsen! I'll review this later tonight and give feedback and hopefully can get it merged in quickly.

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name: "Test using kind and chart-testing tool"
+on: [pull_request, push]
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetesVersion: ["v1.13.12", "v1.17.0"]
```
Comment (zhill) [inline .github/workflows/test.yaml]: I'm curious, why these two specific versions?

Comment (davidkarlsen) [reply .github/workflows/test.yaml]: copy&paste from my current setup - the values are based on latest available release, and the current version of our cluster 1.13.x)

**Comment by davidkarlsen** [comment]
For more info on the tooling: https://github.com/helm/chart-testing - the action is just a wrapper around it.
You can see some logs in our repo here: https://github.com/evryfs/helm-charts/actions

**Comment by davidkarlsen** [comment]
And the output for this PR: https://github.com/davidkarlsen/anchore-charts/commit/5f72ea2d9996401bceedda777482618679bc00b6/checks?check_suite_id=375113058

**Comment by zhill** [comment]
@davidkarlsen do you mind if I squash-merge it?

**Comment by davidkarlsen** [comment]
👍 cleaner to squash it!

**Pull request merged**